packages/pre-runner/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.12.2",
   "description": "This package is part of Scramjet Transform Hub. The package identifies the sequences and returns the information to back STH.",
   "scripts": {
-    "build:docker": "docker build -t ${SCRAMJET_INT_REPO_PATH}scramjetorg/pre-runner:$npm_package_version .",
+    "build:docker": "docker build -t scramjetorg/pre-runner:$npm_package_version .",
```

Comment:
```suggestion
    "build:docker": "docker build -t scramjetorg/pre-runner:$(jq -r .version package.json) .",
```

Reply:
My mistake, I don't know why I had in my head that it was about the node version.